huaweicloud/services/er/resource_huaweicloud_er_instance.go

```diff
@@ -213,6 +247,15 @@ func resourceInstanceCreate(ctx context.Context, d *schema.ResourceData, meta in
 		return diag.FromErr(err)
 	}
 
+	if tags, ok := d.GetOk("tags"); ok {
+		for k, v := range tags.(map[string]interface{}) {
+			err := addResourceTags(createInstanceClient, "instance", id.(string), k, v.(string))
+			if err != nil {
+				return diag.Errorf("error setting tags to the ER instance: %s", err)
```

Comment:
The err message from function addResourceTags has already been defined, you can just return diag.FromErr(err)

Reply:
done.